daemon/mgr/container_stats.go

```diff
@@ -23,19 +32,39 @@ func (mgr *ContainerManager) StreamStats(ctx context.Context, name string, confi
 	}
 
 	outStream := config.OutStream
-	if (!c.IsRunningOrPaused() || c.IsRestarting()) && !config.Stream {
-		return json.NewEncoder(outStream).Encode(&types.ContainerStats{
-			Name: c.Name,
-			ID:   c.ID,
-		})
+	if !c.IsRunningOrPaused() {
+		return errors.New("can only stats running or paused container")
+	}
+
+	var preCPUStats *types.CPUStats
+
+	wrapContainerStats := func(timestamp time.Time, metric *cgroups.Metrics) (*types.ContainerStats, error) {
+		stats := toContainerStats(timestamp, c, metric)
+		systemCPUUsage, err := getSystemCPUUsage()
+		if err != nil {
+			return nil, err
+		}
+		stats.PrecpuStats = preCPUStats
+		stats.CPUStats.SyetemCPUUsage = systemCPUUsage
+		preCPUStats = stats.CPUStats
+		networkStat, err := mgr.NetworkMgr.GetNetworkStats(c.NetworkSettings.SandboxID)
+		if err != nil {
+			//FIXME(ZYecho): not throw error when not found sandbox
```

Comment:
you can comment  `--net=none` here for the record. it might be not an issue actually.